### PR TITLE
Adding target to page links

### PR DESCRIPTION
**Pull request by ajolo2ky**
Added this so you can specify "target:" in the page's front matter, (i.e. target: "_blank")

### CODE REVIEW

**Comment by Lednerb** [comment]
Thx for your contribution!

Maybe you can add an empty `target: ""` with a comment to the `link` archetype?

Otherwise, I will add this and merge tomorrow.

**Comment by ajolo2ky** [comment]
Adding in link and page

File: layouts/partials/topnav.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
         <ul class="topnav">
             {{ range where (where .Site.Pages ".Type" "page") ".IsPage" true }}
                 {{ if and (isset .Params "link") (ne .Params.link "") }}
-                    <li><a href="{{ .Params.link | relURL }}">{{ .Title }}</a></li>
+                    <li><a href="{{ .Params.link | relURL }}" target="{{ .Params.target }}">{{ .Title }}</a></li>
```
Comment (Lednerb) [inline layouts/partials/topnav.html]: Please add this to the `link` post type, too.

File: archetypes/link.md [hunk 2]
```diff
@@ -9,4 +9,5 @@ tags: []
 
 # Set your external url
 link: "https://github.com/Lednerb/gohugo-theme-bilberry"
+target: "" #_blank|_self|_parent|_top|framename
```
Comment (Lednerb) [inline archetypes/link.md]: This option is not reflected in the `link` post type, yet.

**Comment by Lednerb** [comment]
Hello again, I'm working on the merge.

Just rethought about the `link` post type... I'm sure it's not necessary there because a `link` should always link to another page via `target="_blank"` so the original website stays in the background.

How do you use this `target` feature with the `page` post type?
If you set the `link` parameter you will be redirected to the given link and if you want a static `page`, e.g. for the impress or as a contact page you wouldn't use the `link` property...

**Comment by ajolo2ky** [comment]
Hi, I was using it for some external pages, thus the need for target blank
and for adding a Home link in the top bar, so it should stay in the same
window.

**Comment by Lednerb** [comment]
The change is now implemented in the `master` branch.

**Pull request merged**